### PR TITLE
Enable the deletion of hospitals for a staff member

### PR DESCRIPTION
**Pull request by feeds**

### CODE REVIEW

**Comment by feeds** [comment]
@Baschdl I found the branch, this should dolve the issue :)

File: backend/apps/accounts/views.py [hunk 1]
```diff
@@ -209,6 +210,16 @@ def change_hospital_approval(request,uuid):
         send_mails_for(h)
     return HttpResponseRedirect('/accounts/approve_hospitals')
 
+@login_required
+@staff_member_required
+def delete_hospital(request,uuid):
+    h = Hospital.objects.get(uuid=uuid)
+    name = h.user
+    h.delete()
+    text = format_lazy(_("Du hast die Instiution mit user '{name}' gelöscht."), name=name)
```
Comment (Baschdl) [inline backend/apps/accounts/views.py]: "Instiution"

**Review by Baschdl — APPROVED** [review]
Code looks good

**Review by kevihiiin — APPROVED** [review]
Code & website look good

**Pull request merged**